src/lib.rs

```diff
@@ -214,7 +214,7 @@ pub mod saveload;
 /// Implementations and structures related to bitsets.
 ///
 /// Normally used for `Join`s and filtering entities.
-pub mod bitset;
+mod bitset;
```

Comment:
Did you verify that there is no uses for this? (In benchmarks and examples as those are not automatically tested afaik).

Reply:
@WaDelma There's no way that there could be usages.  This module only contains trait implementations and nothing else.  The docs for it don't generate anything except for the doc comments.